content/en/blog/_posts/2023-07-24-mixed-version-proxy-alpha.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: blog
+title: "Kubernetes 1.28: Upgrade Kubernetes more safely by enabling Mixed Version Proxy (Alpha)"
```

Comment:
How about:
```suggestion
title: "Kubernetes 1.28: A New (alpha) Mechanism For Safer API Upgrades"
```

Does that work? It's only a suggestion.

Reply:
Should we include the "Mixed Version Proxy" also in the title, like: `Kubernetes 1.28:  Mixed Version Proxy - A New (alpha) Mechanism For Safer API Upgrades` or is that too long? 

Also should we use `API upgrades` or `cluster upgrades` here?

Reply:
You can pick. But you don't need Mixed Version Proxy in the title.

Reply:
Updated to use `Kubernetes 1.28: A New (alpha) Mechanism For Safer Cluster Upgrades`